dev-games/ogre/ogre-1.12.9-r1.ebuild

```diff
@@ -54,6 +41,11 @@ RDEPEND="
 	cg? ( media-gfx/nvidia-cg-toolkit )
 	egl? ( media-libs/mesa[egl] )
 	freeimage? ( media-libs/freeimage )
+	gles2? (
+		dev-util/glsl-optimizer
+		dev-util/hlsl2glslfork
```

Comment:
Are these two packages really mandatory for gles2 support? I removed these libraries from RDEPEND and ogre still compiles fine with gles2 USE.

Reply:
OH DAMMIT! You are right.

For all versions in 1.12.x both got included and used unconditionally whenever GLES2 was enabled.

And now that I finally have working ebuilds, they fixed it. :angry: 

...*sigh*... I should be glad, of course, and will reverse those dependencies.

However, as configure states:
````
-----------------------------------------------------------------------------
-- The following OPTIONAL packages could NOT be located on your system.
-- Consider installing them to enable more features from this software.
+ Softimage: Softimage SDK needed for building XSIExporter <FALSE>
+ GLSL Optimizer: GLSL Optimizer <http://github.com/aras-p/glsl-optimizer/>
+ HLSL2GLSL: HLSL2GLSL <http://hlsl2glslfork.googlecode.com/>
-----------------------------------------------------------------------------
````

Both tools _are_ usedful. So I really would like to have them included, although with their own individual USE flags, of course. Do you thik that would be in order?

Reply:
Of course they would need their own USE flags, and they probably have their usefulness on full opengl rendering system thus not reliant on gles2 flag. But I wouldn't introduce additional packages only being specifically used for enabling optional features. If anything, python support would be a more worthy feature.

Reply:
Well, they allow you to import DX9 shaders and have them optimized. That's very useful. 

Actually both hlsl2glsl.h and glsl_optimizer.h are only used in RenderSystems/GLES2.